features/step_definitions/basic_steps.rb

```diff
@@ -113,6 +113,11 @@
   sleep seconds.to_i.seconds
 end
 
+And (/^I wait for ajax$/) do
```

Comment:
My preference would be "I wait for ajax to complete"
or even "I wait for all ajax requests to complete"

My goal is to communicate the intent that we're waiting for things to _complete_